docs/src/pages/customization/themes/CustomStyles.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+import {
+  createMuiTheme,
+  makeStyles,
+  createStyles,
+  Theme as AugmentedTheme,
```

Comment:
This type-checked locally without the alias. Was this aliased for testing only or what was the reason here?

Reply:
When I didn't alias it, babel failed to remove it.
https://circleci.com/gh/mui-org/material-ui/102642?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Right, the babel plugin only supports isolated modules in which module augmentation isn't possible.